src/emulator/controller.ts

```diff
@@ -370,7 +370,7 @@ export async function startAll(options: Options, showUI: boolean = true): Promis
 
   if (shouldStart(options, Emulators.HUB)) {
     const hubAddr = await getAndCheckAddress(Emulators.HUB, options);
-    const hub = new EmulatorHub({ projectId, ...hubAddr });
+    const hub = new EmulatorHub({ projectId: projectId!, ...hubAddr });
```

Comment:
Why do you need to assert the value here? rather than doing this, I'd rather the code check that it exists first and fail out otherwise. Doesn't look like it was _required_ before...